jobs/JREGIONAL_RUN_PYTHON_ENSMEAN

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# This script runs the adb_graphics create_graphics driver for creating
+# PNG figures and zipping them for dissemination to the web.
```

Comment:
This comment could use an update.

Reply:
Will update the comments. Was trying to follow the script for the python plotting but forgot to make changes accordingly. Thanks for noticing this!